index.html

```diff
@@ -9,16 +9,187 @@
 </head>
 <body>
     <h1>단어 게임</h1>
+    <h3>score: <p style="display:inline;" id="score"></p></h3>
+    <p id="word"></p>
+    <div class="text">
+    <input id='myanswer' onchange='inputAnswer()' value=""/>
+    </div>
+    <br>
+    <br>
+    <div class="button">
+        <button id="start" onclick="startButton()">게임 시작</button> &nbsp;&nbsp;
+        <button id="end" onclick="endButton()">게임 끝내기</button>
+    </div>
+    <script>
+        const _words = ["apple",
+        "banana",
+        "cat",
+        "dog",
+        "elephant",
+        "fish",
+        "grape",
+        "horse",
+        "ice cream",
+        "jungle",
+        "kiwi",
+        "lion",
+        "monkey",
+        "nut",
+        "orange",
+        "peach",
+        "quail",
+        "rabbit",
+        "strawberry",
+        "tiger",
+        "unicorn",
+        "violet",
+        "whale",
+        "xylophone",
+        "yak",
+        "zebra",
+        "airplane",
+        "ball",
+        "car",
+        "desk",
+        "eleven",
+        "flower",
+        "guitar",
+        "hat",
+        "ice",
+        "juice",
+        "kite",
+        "lemon",
+        "mouse",
+        "nose",
+        "orange",
+        "pencil",
+        "queen",
+        "rainbow",
+        "snake",
+        "tree",
+        "umbrella",
+        "violin",
+        "water",
+        "xylophone",
+        "yoyo",
+        "zebra",
+        "angel",
+        "baby",
+        "cherry",
+        "dolphin",
+        "eagle",
+        "fairy",
+        "ghost",
+        "honey",
+        "island",
+        "jacket",
+        "king",
+        "leopard",
+        "moon",
+        "nest",
+        "owl",
+        "panda",
+        "quilt",
+        "rain",
+        "sun",
+        "tulip",
+        "unicorn",
+        "vase",
+        "watermelon",
+        "xylophone",
+        "yarn",
+        "zeppelin",
+        "apple",
+        "banana",
+        "cat",
+        "dog",
+        "elephant",
+        "fish",
+        "grape",
+        "horse",
+        "ice cream",
+        "jungle",
+        "kiwi",
+        "lion",
+        "monkey",
+        "nut",
+        "orange",
+        "peach",
+        "quail",
+        "rabbit",
+        "strawberry",
+        "tiger",
+        "unicorn",
+        "violet",
+        "whale",
+        "xylophone",
+        "yak",
+        "zebra"];
 
-    <!-- 점수판 -->
+        //score 변수 선언
+        var score = 0;
+        //n=startbutton이 눌렸는지 안 눌렸는지를 구분해줄 변수
+        var n=0;
```

Comment:
```suggestion
        var n = 0;
```
처럼 변수명과 대입 연산자, 값 사이에는 공백을 한 칸 주는게 관용적인 것 같아요.
모던 자바스크립트에 [코드스타일](https://ko.javascript.info/coding-style) 챕터를 한 번 읽어보는게 좋을 것 같습니다.